language-design.md

```diff
@@ -62,7 +186,21 @@ the commenters?
 
 **Response**
 
+The conflict set up by the article and the comments is that professional programmers require robust tools with high specificity while everyone else may prefer languages that are higher-level abstractions. It seems that the premise of the article relies on the idea that most people should be able to understand code and that programming languages are not meeting their goals of being readable since people who do not program professionally cannot comprehend code. We disagree with this notion and side with the substance of the majority of the commenters' arguments. Programming should be something that is learned is not entirely intuitive to every English speaker. Just as English would not be intuitive to a Latin scholar, even though English is partially derived from Latin, programming languages are not intuitive to English speakers, even though many programs and languages are based in English. We believe that it is naive to think that programming languages can become intuitive to any English speaker, and that programming languages should be learned just like French and Spanish. Just like how there is overlapping grammar and vocabulary between Latin languages that ease fluency between speakers of those languages, we also believe this is how programmers quickly develop an ease of fluency between C derived languages. 
```

Comment:
I really like your last point about fluency in English not translating to fluency in a programming language.